src/app.ts

```diff
@@ -107,34 +110,36 @@ const instanceLauncher = new InstanceLauncher({
     redisClient,
 });
 
-async function startPooling() {
-    logger.info('Start pooling..');
+const jobManager = new JobManager({
+    queueRedisOptions: {
+        host: '127.0.0.1',
```

Comment:
This should be the config redis host and port, not hardcoded to 127.0.0.1

Reply:
Thank you for noticing this, I updated the code.